src/components/sections/communities/overview/Sidebar.tsx

```diff
@@ -30,19 +30,36 @@ export default function Sidebar(): JSX.Element {
   };
 
   const scoreboardLink = hasCurrentCommunity ? `/communities/${currentCommunity.slug}/scoreboard` : "";
+  const learningMaterialsLink = hasCurrentCommunity ? `/communities/${currentCommunity.slug}/learning-materials` : "";
   const mainLink = hasCurrentCommunity ? `/communities/${currentCommunity.slug}` : "";
```

Comment:
consider optimizing these links by using a `useMemo` hook
```suggestion
  const links = useMemo(() => {
    if (!hasCurrentCommunity)
      return {
        scoreboardLink: "",
        learningMaterialsLink: "",
        mainLink: "",
      };
    const { slug } = currentCommunity;
    return {
      scoreboardLink: `/communities/${slug}/scoreboard`,
      learningMaterialsLink: `/communities/${slug}/learning-materials`,
      mainLink: `/communities/${slug}`,
    };
  }, [hasCurrentCommunity, currentCommunity]);
```